docs/TROUBLESHOOTING.md

```diff
@@ -248,6 +248,65 @@ In a deploy using the [Hub and Spoke](https://cloud.google.com/architecture/secu
 
 This is a transient error and the deploy can be retried. Wait for at least a minute and retry the deploy.
 
+
+### Error: Unknown project id on 4-project step context
+
+**Error message:**
+
+```text
+Error 400: Unknown project id: 'prj-<business-unity>-<environment>-sample-base-<random-suffix>', invalid
+```
+
+**Cause:**
+
+When you try to run 4-projects step without requesting additional project quota for **project service account created in 0-bootstrap step** you may face the error above, even after the project quota issue is resolved, due to an inconsistency in terraform state.
+
+**Solution:**
+
+- Make sure you [have requested the additional project quota](#project-quota-exceeded) for the **project SA e-mail** before the following steps.
+
+You will need to mark some resources as tainted in order to trigger the proper missing project's recreation to fix the inconsistent terraform state by following the next steps.
+
+1. In your local, go to the path that the error is being reported.
+
+   For example, if the unknown project ID is `prj-bu1-p-sample-base-abcd`, you should go to ./gcp-projects/business_unit_1/production (business_unit_1 due to `bu1` and production due to `p`).
```

Comment:
Linking to the docs/or white paper for the naming conventions may be worthwhile here if there is a suitable link.

Reply:
fixed